Dockerfile

```diff
@@ -1 +1 @@
-FROM drydock-prod.workiva.net/workiva/dart_build_image:3 as build
```

Comment:
leftover from previous release process, now that `scip-dart` is published to pub.dev, publishing will be done manually (until we can figure out how to automate it)